rest.insomnia.client.yml

```diff
@@ -0,0 +1,64 @@
+app-id: rest.insomnia.client
+runtime: org.freedesktop.Sdk
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '20.08'
+rename-desktop-file: insomnia.desktop
+rename-icon: insomnia
+separate-locales: false
+command: insomnia
+finish-args:
+  - --socket=x11
+  - --socket=wayland
```

Comment:
```suggestion
```

I assume it uses xwayland on Wayland.

Reply:
Xwayland is a fallback for apps that don't support wayland, and has issues (e.g.: no proper scaling support).